### PR TITLE
Update peaq network name, RPC

### PR DESCRIPTION
**Pull request by lavish0000**

### CODE REVIEW

**Comment by lavish0000** [comment]
@ligi  Can you please review this?

File: _data/chains/eip155-3338.json [hunk 1]
```diff
@@ -17,11 +19,6 @@
   "chainId": 3338,
   "networkId": 3338,
   "explorers": [
-    {
-      "name": "Polkadot.js",
-      "url": "https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpeaq.api.onfinality.io%2Fpublic-ws#/explorer",
```
Comment (AngelicaOvesdotter) [inline _data/chains/eip155-3338.json:22]: - [ ]

**Comment by lavish0000** [comment]
@ligi  Can you please review this?

**Comment by AngelicaOvesdotter** [comment]
> @ligi Can you please review this?
image

**Comment by AngelicaOvesdotter** [comment]
image

**Comment by lavish0000** [comment]
oh My bad 😅, @AngelicaOvesdotter . Can you please review and merge?

**Comment by ligi** [comment]
If people constantly ping the PR will be merged delayed - if you are i a hurry pay some funds to lists.eth

**Review by ligi — APPROVED** [review]

**Pull request merged**